Valigator.Newtonsoft.Json/ValigatorJsonPropertyHandler.cs

```diff
@@ -61,12 +61,17 @@ public override void WriteProperty(JsonWriter writer, JsonSerializer serializer,
 				return new ValigatorJsonPropertyHandler<TObject, TDataValue>(null, null);
 
 			var objParameter = Expression.Parameter(typeof(TObject), "obj");
-			var propertyExpression = Expression.Property(objParameter, property);
+
+			var propertyParameter = Expression.Constant(property);
```

Comment:
Once again, call the underlying getters and setters instead of relying on it being a property.